pkg/controller/node/node_controller.go

```diff
@@ -451,6 +430,26 @@ func (r *ReconcileNode) createNsxClients() (*NsxClients, error) {
 	return &nsxClients, nil
 }
 
+func (r *ReconcileNode) isTagDisabled() (bool, error) {
+	ncpInstall := &operatorv1.NcpInstall{}
+	watchedNamespace := r.status.GetOperatorNamespace()
+	if watchedNamespace == "" {
+		log.Info(fmt.Sprintf("NodeController can only watch a single namespace, defaulting to: %s",
+			operatortypes.OperatorNamespace))
+		watchedNamespace = operatortypes.OperatorNamespace
+	}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: operatortypes.NcpInstallCRDName, Namespace: watchedNamespace}, ncpInstall)
```

Comment:
How about put ncpInstall in sharedInfo, so you don't need to get it again in node_controller

Reply:
Yes the code is redundant, will try to improve it.

Reply:
done.